src/translations/es.json

```diff
@@ -0,0 +1,62 @@
+{
+  "camera": {
+    "all_cameras": "Todas las Vámaras",
+    "cameras": "Cámaras"
+  },
+  "climate": {
+    "all_climates": "Todos los Aires Acondicionados",
+    "climates": "Aires Acondicionados"
+  },
+  "cover": {
+    "all_covers": "Todas las Cortinas",
+    "covers": "Cortinas"
+  },
+  "fan": {
+    "all_fans": "Todos los Ventiladores",
+    "fans": "Ventiladores"
+  },
+  "generic": {
+    "all": "Todo",
+    "areas": "Áreas",
+    "busy": "Ocupado",
+    "good_afternoon": "Buenas tardes",
+    "good_evening": "Buenas noches",
+    "good_morning": "Buenos días",
+    "hello": "Hola",
+    "home": "Inicio",
+    "miscellaneous": "Miscellaneous",
+    "numbers": "Números",
+    "off": "Apagar",
+    "on": "Encender",
+    "open": "Abrir",
+    "unclosed": "Sin Cerrar"
```

Comment:
These are about states, not about verbs.
Like: is off, is on, is open, is not closed.